src/empty.js

```diff
@@ -8,8 +8,9 @@ var _isString = require('./internal/_isString');
 /**
  * Returns the empty value of its argument's type. Ramda defines the empty
  * value of Array (`[]`), Object (`{}`), String (`''`), and Arguments. Other
- * types are supported if they define `<Type>.empty` and/or
- * `<Type>.prototype.empty`.
+ * types are supported if they define `<Type>.empty`,
+ * * `<Type>.prototype.empty` or implement the
```

Comment:
is there an extra `* ` at the beginning of this line?

Reply:
@kedashoe yep it was. Fixed in additional commit. Thanks.